services/viva-ms/lambdas/generateRecurringCaseHtml.js

```diff
@@ -100,10 +51,11 @@ export async function main(event, context) {
     return false;
   }
 
+  let answers = caseItem.forms[vivaCaseSSMParams.recurringFormId].answers;
   if (closedCases.length > 0) {
     const [closedCase] = closedCases.sort((caseA, caseB) => caseB.updatedAt - caseA.updatedAt);
 
-    setChangedCaseAnswerValues(
+    answers = getChangedCaseAnswerValues(
```

Comment:
`answers` might be able to be renamed `changedAnswerValues` or `dirtyAnswers` to enlight that they are not the same. :)

Reply:
Done